content/en/docs/faas/_index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Functions as a Service
+description: >-
+  OpenTelemetry supports various methods of monitoring Functions as a Service
+  provided by different cloud vendors
+weight: 2
```

Comment:
Please update the weight in a way that FaaS docs comes after the instrumentation & collector

Reply:
How long does it take the preview to update?